frontend/test/components/inputElement.spec.ts

```diff
@@ -61,4 +64,23 @@ describe("InputElement", () => {
     expect(emitted()["update:modelValue"]).toHaveLength(1)
     expect(emitted()["update:modelValue"]).toEqual([["a"]])
   })
+
+  it("renders an error message if input is not valid", async () => {
+    renderComponent({
+      id: "Testfeld",
+      type: InputType.DATE,
+      modelValue: "2024-02-10",
```

Comment:
Maybe +10 years or something? Otherwise next year that test will fail 😉 No biggie though, then we can adjust it of course